src/__tests__/tags.test.ts

```diff
@@ -0,0 +1,51 @@
+import {DocumentBuilder, run} from './__helpers__/run';
+
+const name = 'tags';
+describe('tags rendering', () => {
+    it('renders tag and toc', async () => {
```

Comment:
I think these test names poorly reflect on what's actually being tested. This PR is implementing a mechanism to override page headings and URLs when specific vendor extensions are used. I could suggest naming the test suite something along the lines of `Identity of pages derived from spec tags` with cases named like `can be overriden using x-slug to define a custom URL`. Anyway, I'm open to suggestions here.

Reply:
I've split test cases